fe/src/assets/css/tailwind.css

```diff
@@ -29,3 +29,62 @@ body {
   white-space: nowrap;
   width: 100%;
 }
+/* Paginator */
+.paginator-arrow-heads {
+  @apply flex items-center;
+}
+.paginator-icon {
+  @apply items-center justify-center rounded-md border border-transparent leading-tight;
+}
+.paginator-icon.disabled {
+  @apply cursor-not-allowed text-neutral-500;
+}
+.paginator-icon.enabled {
+  @apply cursor-pointer hover:text-primary-10;
+}
+.paginator-icon.arrow {
+  @apply ml-1 flex h-4 w-4;
+}
+.paginator-icon.arrow .icon {
+  @apply h-full w-full;
+}
+.paginator-icon.ellipsis {
+  @apply ml-2 flex h-6 w-6;
+}
+
+.paginator-number {
+  @apply ml-2 flex h-6 w-6 cursor-pointer items-center justify-center rounded-md border border-transparent text-center text-xs leading-tight  hover:border-primary-10;
+}
+.paginator-number.active {
+  @apply bg-primary-10 text-white;
+}
+.paginator-number.not-active {
+  @apply text-neutral-500 hover:text-primary-10;
+}
```

Comment:
No need to create a custom class here in tailwind.css.
Just apply these classes directly to your paginator component elements.
```suggestion
```